contracts/ERC721RExample.sol

```diff
@@ -122,7 +135,7 @@ contract ERC721RExample is ERC721A, Ownable {
     }
 
     function toggleRefundCountdown() public onlyOwner {
-        refundEndTime = block.timestamp + refundPeriod;
+        refundEndTime = block.number + refundPeriod;
```

Comment:
do we want to rename `refundEndTime` to something like `refundEndBlockNumber`?

Reply:
Sure, that makes sense.